README.md

```diff
@@ -15,3 +15,33 @@ Resolver is a DataHub microservice for resolving datapackage URLs into more huma
 # Run server
 
 `python server.py`
+
+
+## API
+
+### Get information about userid and packageid 
```

Comment:
I don't think this is what it does. 
something like "resolves username to userid" would be better. Or maybe you can think something better